### DISCUSSION TITLE
Welcome to redis-py-cluster Discussions!

### DISCUSSION THREAD

**Discussion by Grokzen** [Grokzen/redis-py-cluster#422] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zhaoyi2** [comment]


**Reply by holmboe** [reply]
It seems that you want to open a new topic for discussion? Hit the "New discussion" button.

![image](https://user-images.githubusercontent.com/308877/104614199-536b1600-5688-11eb-953e-c3dd7d6e4d35.png)
